integration-tests/endpoint.go

```diff
@@ -9,18 +9,20 @@ type EndpointInfo struct {
 	Protocol string
 	ListenAddress string
 	CloseTimestamp string
+	Originator string
 }
 
 func NewEndpointInfo(line string) (*EndpointInfo, error) {
 	parts := strings.Split(line, "|")
 
-	if len(parts) != 4 {
-		return nil, fmt.Errorf("invalid gRPC string for endpoint info: %s", line)
+	if len(parts) != 5 {
+		return nil, fmt.Errorf("Invalid gRPC string for endpoint info: %s", line)
 	}
 
 	return &EndpointInfo {
 		Protocol: parts[1],
 		ListenAddress: parts[2],
 		CloseTimestamp: parts[3],
+		Originator: parts[4],
```

Comment:
Can we turn Originator into a `ProcessInfo`? I appreciate the string format is different but it contains the same data and would make the assertion in the test a little clearer 

I guess the ideal would be to store the information in the same format in the Process bucket as well as in the Endpoints bucket so they can be parsed in the same way

Reply:
Done